modules/interconnect_attachment/variables.tf

```diff
@@ -37,6 +37,7 @@ variable "region" {
 variable "interconnect" {
   type        = string
   description = "URL of the underlying Interconnect object that this attachment's traffic will traverse through."
+  default     = ""
```

Comment:
@rhariady pz set the default value to null

Reply:
set it to null will result in error, as reported by @rnfawcett in https://github.com/terraform-google-modules/terraform-google-cloud-router/issues/7#issuecomment-877521002